src/entities/day-status-record.ts

```diff
@@ -0,0 +1,32 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+import {Enum, MaxLength, Property, Required} from "@tsed/common";
+import {DayStatusRecordType} from "../enums";
+
+@Entity("day_status_record")
+export class DayStatusRecord {
+
+  @PrimaryGeneratedColumn()
+  @Property()
+  id: number;
```

Comment:
Better to create a `BaseEntity` class with this field there. No need to duplicate it across all entities.